src/Symfony/Bridge/Doctrine/Form/ChoiceList/ORMQueryBuilderLoader.php

```diff
@@ -93,6 +93,13 @@ public function getEntitiesByIds($identifier, array $values)
             $values = array_values(array_filter($values, function ($v) {
                 return (string) $v === (string) (int) $v;
             }));
+        } elseif ('guid' === $metadata->getTypeOfField($identifier)) {
+            $parameterType = Connection::PARAM_INT_ARRAY;
+
+            // Like above, but we just filter out empty strings.
+            $values = array_values(array_filter($values, function ($v) {
+                return (string) $v !== '';
```

Comment:
But validation can already be done via [constraint](http://symfony.com/doc/current/reference/constraints/Uuid.html)
The purpose here is allowing empty strings for non-required uuid fields